packages/next/src/server/dev/hot-reloader-webpack.ts

```diff
@@ -1594,4 +1594,6 @@ export default class HotReloaderWebpack implements NextJsHotReloaderInterface {
         })
       })
   }
+
+  public close() {}
```

Comment:
Why don't we close them for webpack as well?

Reply:
The webpack implementation doesn't currently track the websocket clients, so it requires extra code. I was in a rush to get something up because this was blocking me on local tasks. I can add it.

Reply:
there's close handling here we just don't have wired up it seems https://github.com/vercel/next.js/blob/e85b4270433650b0b26d62bbbd883eb7a5afbdfc/packages/next/src/server/dev/hot-middleware.ts#L247-L253